pkg/controller/backup_controller.go

```diff
@@ -42,6 +42,7 @@ import (
 	arkv1client "github.com/heptio/ark/pkg/generated/clientset/typed/ark/v1"
 	informers "github.com/heptio/ark/pkg/generated/informers/externalversions/ark/v1"
 	listers "github.com/heptio/ark/pkg/generated/listers/ark/v1"
+	"github.com/heptio/ark/pkg/util"
```

Comment:
Let's not have a `util` package. `util/errors` aliased to `arkerrors` or something would be better imho

Reply:
fixed

Reply:
yes, very much lets not have a generic `util` package.